1_fetch.R

```diff
@@ -320,5 +320,11 @@ p1_targets_list <- list(
                                                "TotDASqKM", "DivDASqKM", "SLOPE", 
                                                "LakeFract", "SurfArea")), 
              deployment = "main"
-  )
+  ),
+  
+  ##screening out NA and -9998, -9999 values
+  tar_target(p1_feature_vars_conus_screen,
+             p1_feature_vars_conus %>%
+               na.exclude() %>%
+               filter_if(is.numeric, all_vars(. > -990)))
```

Comment:
Did you check that this only filters out the -9999 and -9998 rows? I don't think there are any attributes whose real values are < -990, though.

Reply:
yes, I looked at all the values.  There are some attributes that have negative values only up to about -750.

Reply:
Attributes with NAs were:  

[1] "CAT_STREAMRIVER" "CAT_ARTIFICIAL"  "CAT_CANALDITCH"  "CAT_CONNECTOR"   "CAT_PIPELINE"    "ACC_STREAMRIVER"
 [7] "ACC_ARTIFICIAL"  "ACC_CANALDITCH"  "ACC_CONNECTOR"   "ACC_PIPELINE"    "TOT_STREAMRIVER" "TOT_ARTIFICIAL" 
[13] "TOT_CANALDITCH"  "TOT_CONNECTOR"   "TOT_PIPELINE"    "CAT_STRM_DENS"   "ACC_STRM_DENS"   "TOT_STRM_DENS"